src/components/network-modification-tree.js

```diff
@@ -214,3 +332,11 @@ const NetworkModificationTree = ({ treeModel, openModificationDrawer }) => {
 };
 
 export default NetworkModificationTree;
+
+NetworkModificationTree.propTypes = {
+    treeModel: PropTypes.object.isRequired,
+    drawerNodeEditorOpen: PropTypes.bool.isRequired,
```

Comment:
See previous comment about removing drawerNodeEditorOpen prop and storing it to redux store.

Reply:
@AbdelHedhili What do you think about it ? We can discuss it tomorrow

Reply:
done